projects/vdk-plugins/airflow-provider-vdk/tests/hooks/test_vdkhook.py

```diff
@@ -1,9 +1,48 @@
 # Copyright 2021 VMware, Inc.
 # SPDX-License-Identifier: Apache-2.0
+import logging
+import unittest
+from unittest import mock
+
+import requests_mock
 from vdk_provider.hooks.vdk import VDKHook
 
 
-def test_dummy():
-    hook = VDKHook("conn_id", "job_name", "team_name")
+log = logging.getLogger(__name__)
+
+
+# Mock the `conn_sample` Airflow connection
+@mock.patch.dict(
+    "os.environ", AIRFLOW_CONN_CONN_VDK="http://https%3A%2F%2Fwww.vdk-endpoint.org"
+)
+class TestVDKHook(unittest.TestCase):
```

Comment:
Unless you re-use something from unittest.TestCase, you should not need to inherit from it,

Reply:
noted